tools/runners/test262-harness.py

```diff
@@ -910,6 +911,13 @@ def list_includes(self, tests):
 
 
 def main():
+    (out_stream, err_stream) = (sys.stdout.buffer, sys.stderr.buffer) \
+                                if sys.version_info.major >= 3 else (sys.stdout, sys.stderr)
+    # For tty using native encoding, otherwise (pipe) use 'utf-8'
```

Comment:
@akosthekiss how about move these code into util， they can be shared across different tools